### PR TITLE
Sayali/userlogin

### PR DESCRIPTION
**Pull request by sayalisheth**

### CODE REVIEW

File: src/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// {
```
Comment (amitk) [inline src/.eslintrc.json]: @sayalisheth remove this file.

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <>
+
```
Comment (amitk) [inline src/App.js]: @sayalisheth remove extra spaces.

File: src/components/Login.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+return (
+   <div>
+      <div className="card col-sm-4">
+          <div className="card__container">
+              <div>
+                <h1 className="card__title">Login</h1>
+              </div>
+              
+              <div className="row">
+                <input type="text" className="add__input" placeholder="Username" />
```
Comment (amitk) [inline src/components/Login.js]: you can use reactstrap elements. https://reactstrap.github.io/

File: src/.eslintrc.json (hunk 1 above)
Comment (sayalisheth) [reply src/.eslintrc.json]: Done

File: src/App.js (hunk 2 above)
Comment (sayalisheth) [reply src/App.js]: Done

File: src/components/Login.js (hunk 3 above)
Comment (sayalisheth) [reply src/components/Login.js]: Done

File: src/App.js [hunk 4]
```diff
@@ -1,25 +1,17 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+// import 'bootstrap/dist/css/bootstrap.min.css'
```
Comment (amitk) [inline src/App.js]: @sayalisheth remove commented code.

Comment (sayalisheth) [reply src/App.js]: Done

**Review by amitk — APPROVED** [review]

**Pull request merged**